python/mdvtools/project_router.py

```
@@ -163,47 +162,95 @@ def dispatch_request(self, subpath: str, project_id) -> Response:
+                        
+                        
+                        
                 # first determine whether this is allowed
                 # - rather than iterating over (rule, method), we might have
                 # (rule, (method, permissionsFlags))
                 # we can check the request.token (or whatever it is) here...
-                print("options",options)
                 # Check for access level only if specified in options
                 if options and 'access_level' in options:
-                    print("match", match)
+                    logger.info(f"match 'match'")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)